From db4d5d9ecbf9167b45dd5acd8526c739e01e040e Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@rvc.renesas.com>
Date: Wed, 13 Mar 2019 09:58:54 +0700
Subject: [PATCH 1/2] hmi: demos: Encode Preview: change size of display video

Due to limitation of vspfilter(can't handle 2 pipeline with different
size of frame), change resolution of display pipeline as
same as encode capture pipeline.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@rvc.renesas.com>
---
 RZ_GST-demo_encoded-review.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/RZ_GST-demo_encoded-review.sh b/RZ_GST-demo_encoded-review.sh
index 48c23e9..4519eac 100755
--- a/RZ_GST-demo_encoded-review.sh
+++ b/RZ_GST-demo_encoded-review.sh
@@ -14,7 +14,7 @@ fi
 function replay()
 {
 	printf "\e[34mPlaying... $1\e[0m\n"
-	gst-launch-1.0 -e filesrc location=$1 ! qtdemux ! queue ! h264parse ! omxh264dec ! vspfilter ! video/x-raw,width=800,height=480 ! waylandsink qos=false
+	gst-launch-1.0 -e filesrc location=$1 ! qtdemux ! queue ! h264parse ! omxh264dec ! vspfilter ! video/x-raw,format=NV12,width=640,height=480 ! waylandsink qos=false
 }
 
 function record()
--
2.7.4
